docs/inputs.md

```diff
@@ -75,7 +75,13 @@ All rules from Number type also apply.
 
 ### Select
 
-@todo
+A dropdown select. Add options with array:
```

Comment:
you might also provide the map style. Here's my take, for a ClojureScript wrapper: https://leva.mentat.org/#select

Reply:
Not sure if thats useful for general public, also the other Inputs don't have map style either